src/widget/widget.cpp

```diff
@@ -498,6 +498,19 @@ void Widget::onTransferClicked()
     activeChatroomWidget = nullptr;
 }
 
+bool Widget::confirmExecutableOpen(const QFileInfo file)
+{
+    if (file.isExecutable())
```

Comment:
@TheLastProject that stackoverflow post seems to be wrong.

If you look at the Qt5 source code (I also looked at 4.8 and 4.2) it seems to only be checking permissions. The doc also implies that it just check permissions, it never mentions filenames.

![Uploading snapshot5.png . . .]()